python/README.md

```diff
@@ -18,3 +18,20 @@ In order to obtain relevant modules, you should install them from PyPI directly,
 ```
 pip install geoarrow-rust-core
 ```
+
+## Development
+
+To install versions of the package under active development, clone the repo and open the `python` directory:
+
+```
+gh repo clone geoarrow/geoarrow-rs
+cd geoarrow-rs/python
+```
+
+From there you can install the package locally with `pip`:
+
+```
+pip install .
+```
```

Comment:
For local development I'd suggest using maturin directly instead.

```
cd python/core
virtualenv env
source ./env/bin/activate
pip install -U maturin
maturin develop
```

That will build the wheel and install into the current virtualenv. You should note that's a debug build, so it'll run slower though. Use `--release` to build a release build.

Reply:
Tested and now added.